storage-s3-dynamodb/src/test/java/io/delta/storage/utils/ReflectionUtilsSuite.java

```diff
@@ -0,0 +1,55 @@
+/*
+ * Copyright (2021) The Delta Lake Project Authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.delta.storage.utils;
```

Comment:
My bad. I didn't realize the test case is in scala. Is it possible to move these test classes as scala classes to `ReflectionUtilsSuit.scala`.

Reply:
Scala class has some additional symbol un name, I don't remember what exactly. I will try to use them.

Reply:
Ok, Scala inner class pass to all constructor instance of Wrapper class.
In our case Constructor object will look like `public io.delta.storage.utils.ReflectionsUtilsSuite$TestOnlyAWSCredentialsProviderWithHadoopConf(io.delta.storage.utils.ReflectionsUtilsSuite,org.apache.hadoop.conf.Configuration)
`.
 So It will not work with this reflections.